Makefile

```diff
@@ -25,8 +25,9 @@ TEST_COVER_PACKAGES:=go list ./pkg/... | grep -vE "pkg/client" | grep -vE "pkg/t
 
 default: build
 
-docker-push: docker
+docker-push: docker backup-docker
 	docker push "${DOCKER_REGISTRY}/pingcap/tidb-operator:latest"
+	docker push "${DOCKER_REGISTRY}/pingcap/tidb-backup-manager:latest"
```

Comment:
how to version `tidb-backup-manager` image, same as tidb-operator or independently? Is it possible to distribute tidb-backup-manager in the tidb-operator image?

Reply:
`tidb-backup-manager`  is an independent binary and always uses the latest.  tidb-operator use `alpine:3.5` as the basic image, tidb-backup-manager use `pingcap/tidb-enterprise-tools:latest` as the bacis image. So I think it’s not appropriate to put them together.